src/components/dialogs/exit.vue

```diff
@@ -1,6 +1,29 @@
 <template>
   <v-dialog v-model="showModal" max-width="45rem">
-    <v-card>
+    <!-- exit incomplete payment -->
```

Comment:
This code looks OK but I think it would be better to create a second dialog file here.

Separate dialog files would keep the logic out of here and into the caller. That would simplify this file and would probably simplify the caller as well: it would just be a separate conditional, ie. "if A then show D1 else if B then show D2 else ...".

The more simple the code in this project, the better for all of us :)